react/PageBlock/PageBlock.less

```diff
@@ -1,12 +1,6 @@
 @import (reference) "~seek-style-guide/theme";
 
-@classic-ia: ~"only screen and (min-width: 1025px)";
-
 .content {
-  @media @classic-ia {
-    max-width: @grid-container-width + (@grid-gutter-width * 2);
-    margin: 0 auto;
-    padding-left: @grid-gutter-width;
-    padding-right: @grid-gutter-width;
-  }
+  max-width: @grid-container-width;
+  margin: 0 auto;
```

Comment:
Why are removing padding? Won't this push some content to the edges of the screen?